src/ring_theory/local_properties.lean

```diff
@@ -419,9 +418,8 @@ begin
   haveI : is_scalar_tower R R' S' :=
     is_scalar_tower.of_algebra_map_eq' (is_localization.map_comp _).symm,
   let fₐ : S →ₐ[R] S' := alg_hom.mk' (algebra_map S S') (λ c x, ring_hom.map_mul _ _ _),
-  tactic.unfreeze_local_instances,
 
-  obtain ⟨T, hT⟩ := hf,
+  unfreezingI { obtain ⟨T, hT⟩ := hf },
```

Comment:
What type is `hf` here?

Reply:
`f` is some `R ->+* S`, and `hf : f.finite_type`.